cmd/ytdl/main.go

```diff
@@ -159,6 +159,8 @@ func handler(identifier string, options options) {
 		log.Logger = log.Level(zerolog.InfoLevel)
 	}
 
+	ytdl.SetLogger(log.Logger)
```

Comment:
This code operates on the global zerolog logger; it'd be better to write to a local variable then do `SetLogger`.

(I'd comment on line 149, but it won't let me do that in the PR since it's not a part of the change.)

Reply:
Actually, this package uses the global logger anyway, so I guess it doesn't matter.

Reply:
Using the global logger is fine because it's the main package.